### ISSUE TITLE
Steamcommunity.com Some Functionality is Broken due to CSP

### ISSUE THREAD

**Issue by StevenRoddis** [EFForg/https-everywhere#909]
eg. http://steamcommunity.com/market/listings/730/Chroma%20Case

Apparently there is something that appears when you hover over the item image. (Or so I've been told)

```
Refused to load the script 'https://steamcommunity-a.akamaihd.net/public/javascript/jqplot/plugins/jqplot.cursor.min.js'
because it violates the following Content Security Policy directive:
"script-src 'self' 'unsafe-inline' 'unsafe-eval' http://steamcommunity-a.akamaihd.net/ https://api.steampowered.com/ http://www.google-analytics.com https://ssl.google-analytics.com https://www.google.com https://www.gstatic.com https://apis.google.com".
```

https://github.com/EFForg/https-everywhere/blob/master/src/chrome/content/rules/Steam_static.com.xml
is the problematic rule.

Do we have the ability to inject `https://steamcommunity-a.akamaihd.net` etc into the site's CSP response? I know Tampermonkey can. Should we?

Or should I just go disable the rules?

**Comment by StevenRoddis** [comment]
The irony of http and 'unsafe-inline' 'unsafe-eval' in CSP is not lost on me.

**Comment by jsha** [comment]
I don't think we should start editing CSP responses. Let's just disable the rules. If you have time, emailing the Steam folks would be a nice extra step.

**Comment by StevenRoddis** [comment]
Do we have a timeframe for the next HTTPSE release? I just emailed Gabe Newell in hopes they fix it on their end.

**Comment by jsha** [comment]
Probably tomorrow.

**Comment by StevenRoddis** [comment]
I tested in Chrome again, it's the Akamai ruleset doing it. I'm preparing a fix now.

**Comment by StevenRoddis** [comment]
Fixed in #924
